nav2_util/include/nav2_util/simple_action_server.hpp

```diff
@@ -220,39 +282,11 @@ class SimpleActionServer
     return current_handle_->is_canceling();
   }
 
-  void terminate_goals(
+  void terminate_current(
```

Comment:
i think there also needs to be a `terminate_all()` that also terminates all pending goals too for the action server cancelled case.

Reply:
that's right